runtime/appruntime/app/app.go

```diff
@@ -61,6 +62,7 @@ func New(p *NewParams) *App {
 		})
 	}
 	rootLogger := zerolog.New(logOutput).With().Timestamp().Logger()
+	customMetrics := custommetrics.NewManager(cfg.Runtime.AppSlug, cfg.Runtime.EnvCloud, rootLogger)
```

Comment:
I was thinking, we could read a `metrics_enabled` flag from config and pass it to `NewManager` to enable/disable metrics?

Reply:
We've decided to have metrics always enabled. We'll revisit this in the future if use cases where metrics should be disabled come up.